projects/components/src/common/activity-reporter/banner-activity-reporter.spec.ts

```diff
@@ -0,0 +1,113 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component, ViewChild } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { ActivityReporterModule } from './activity-reporter.module';
+import { WidgetFinder } from '../../utils/test/widget-object';
+import { ActivityStatus, ActivityResponse } from './activity-reporter';
+import { BannerActivityReporterComponent } from './banner-activity-reporter.component';
+import { BannerActivityReporterWidgetObject } from '../../utils/test/activity-reporter/banner-activity-reporter.wo';
+import { TranslationService, MockTranslationService } from '@vcd/i18n';
+
+@Component({
+    template: `
+        <vcd-banner-activity-reporter #activityReporter></vcd-banner-activity-reporter>
+    `,
+})
+class TestBannerComponent {
+    @ViewChild('activityReporter', { static: true }) activityReporter: BannerActivityReporterComponent;
```

Comment:
You don't need a hash if there's only one `vcd-banner-activity-reporter` See https://angular.io/api/core/ViewChild#example-2